arviz/utils.py

```diff
@@ -753,3 +767,95 @@ def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
 
     return wrapper
+
+def Calculate_ICS(
+    dataset_dict,
+    scale: Optional[ScaleKeyword] = None,
+    ic: Optional[ICKeyword] = None
+):
+
+    """
+    Helper Function used to execute the loo and waic information criteria
+
+    LOO is leave-one-out (PSIS-LOO `loo`) cross-validation and
+    WAIC is the widely applicable information criterion.
+    Read more theory here - in a paper by some of the leading authorities
+    on model selection dx.doi.org/10.1111/1467-9868.00353
+
+    Parameters
+    ----------
+
+    dataset_dict :  dataset_dict: dict[str] -> InferenceData or ELPDData
+    A dictionary of model names and InferenceData or ELPDData objects
```

Comment:
```suggestion
    dataset_dict :  dict of {str: InferenceData or ELPDData}
        A dictionary of model names and InferenceData or ELPDData objects
```

I modified the type signature to follow numpydoc (see examples in https://pandas.pydata.org/pandas-docs/stable/development/contributing_docstring.html#parameter-types) and added the indent to be consistent and use valid rST. This function is not in the online docs, so wrong rST won't give any problems though